src/app/Main/Body/QueryPage/Parameter/ParameterContent-List.component.tsx

```diff
@@ -20,6 +20,9 @@ const listStyle = css`
     padding: 0;
     margin: 0 4em 0 0;
 `;
+const listHeaderStyle = css`
+    position: relative;
```

Comment:
Can you explain what the bug was? (it is to refresh my css knowledge)

Reply:
The headings of the parameters did not move when scrolling. They were visible until the accoridon disappeared completely from the visible field.  I hope the description is understandable.

Reply:
Sorry, i messed up my comment, i had another question: I wanted to know if the position is set to absolute by default on material-ui ListSubHeader components and if that's the cause.